index.html

```diff
@@ -1052,7 +1052,7 @@
           </ol>
           <div class="note">
             The details of implementing the UI and device selection are left to
-            the user agent; for example it MAY show the user a dialog and allow
+            the user agent; for example it may show the user a dialog and allow
```

Comment:
Similarly, "may" could be replaced with "can", so this would read:

"it can show"